src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/model/UI.java

```diff
@@ -154,16 +152,8 @@ public UI(LanternMainActivity activity, SharedPreferences mPrefs) {
         void runCommand();
     }
 
-    public void setVersionNum(final String latestVersion) {
-
-        this.activity.runOnUiThread(new Runnable() {
-            @Override
-            public void run() {
-                mPrefs.edit().putString("versionNum", latestVersion).commit();
-
-                versionNum.setText(latestVersion);
-            }
-        });
+    public void setVersionNum(final String appVersion, final String lanternVersion) {
+        versionNum.setText(String.format("%s-%s", appVersion, lanternVersion));
     }
```

Comment:
After this is called, the text at the bottom of the side-menu will display the app and library version numbers